docs/install/install-kn.md

```diff
@@ -13,13 +13,33 @@ To install the `kn` Client, you must download the executable binary for your sys
 You must place the executable binary in your system path, and make sure that it is executable.
 
 ## Install `kn` using Go
-1. Check out the [Client repository](https://github.com/knative/client).
-1. Run the command:
+1. Build `kn` requires Go v1.13 or newer. Please install [go](https://golang.org/doc/install) environment firstly.
```

Comment:
I would check gramma here. Maybe something like "Building `kn` requires Go .... You will first need a working Go environment.